packages/perseus/src/widgets/interactive-graphs/graphs/components/movable-point.tsx

```diff
@@ -20,29 +20,29 @@ type Props = {
 const hitboxSizePx = 48;
 
 export const StyledMovablePoint = (props: Props) => {
-    const {state} = useGraphState();
+    const {range, snapStep, markings} = useGraphConfig();
     const hitboxRef = useRef<SVGCircleElement>(null);
     const {point, onMove, color = WBColor.blue} = props;
 
     const {dragging} = useMovable({
         gestureTarget: hitboxRef,
         point,
         onMove,
-        constrain: (p) => snap(state.snapStep, p),
+        constrain: (p) => snap(snapStep, p),
     });
     const pointClasses = `movable-point ${dragging ? "movable-point--dragging" : ""}`;
 
     const [[x, y]] = useTransform(point);
 
-    const [xMin, xMax] = state.range[0];
-    const [yMin, yMax] = state.range[1];
+    const [xMin, xMax] = range[0];
+    const [yMin, yMax] = range[1];
```

Comment:
I see you restructured this in one line in the first file like this:

```ts
   range: [[xMin, xMax], [yMin, yMax]]
```

That feels nice and compact and avoids spreading (no pun intended) out this destructuring of the graph config across multiple lines.